test/integration/smart_answer_flows/uk_benefits_abroad_test.rb

```diff
@@ -496,6 +496,137 @@ class UKBenefitsAbroadTest < ActiveSupport::TestCase
+              add_response :after_jan_2021
+            end
+            should "ask has one of your parents ever lived in an EU country, Norway, Iceland, Liechtenstein or Switzerland?" do
+              assert_current_node :parents_lived_in_eea_or_switzerland?
+            end
+            context "answer yes before January 2021" do
+              setup do
+                add_response :before_jan_2021
+              end
+              should "go to esa going_abroad eea outcome" do
+                assert_current_node :esa_going_abroad_eea_outcome
+              end
+            end
+            context "answer yes after January 2021" do
+              setup do
+                add_response :after_jan_2021
+              end
+              should "go to esa going abroad other outcome" do
+                assert_current_node :esa_going_abroad_other_outcome
+              end
+            end
+            context "no" do
+              setup do
+                add_response :no
+              end
+              should "go to esa going abroad other outcome" do
+                assert_current_node :esa_going_abroad_other_outcome
+              end
+            end
+          end
+          context "answer no" do
+            setup do
+              add_response :no
+            end
+            should "ask has one of your parents ever lived in an EU country, Norway, Iceland, Liechtenstein or Switzerland?" do
+              assert_current_node :parents_lived_in_eea_or_switzerland?
+            end
+            context "answer yes before January 2021" do
+              setup do
+                add_response :before_jan_2021
+              end
+              should "go to esa going_abroad eea outcome outcome" do
+                assert_current_node :esa_going_abroad_eea_outcome
+              end
+            end
+            context "answer yes after January 2021" do
+              setup do
+                add_response :after_jan_2021
+              end
+              should "go to esa going_abroad other outcome" do
+                assert_current_node :esa_going_abroad_other_outcome
+              end
+            end
+            context "answer no" do
+              setup do
+                add_response :no
+              end
+              should "go to esa going_abroad other outcome" do
+                assert_current_node :esa_going_abroad_other_outcome
+              end
+            end
+          end
+        end
+        # context "answer albania" do
+        #   setup do
+        #     add_response "albania"
+        #   end
+        #   should "take you to other outcome" do
+        #     assert_current_node :esa_going_abroad_other_outcome
+        #   end
+        # end
+
+        # context "answer kosovo" do
+        #   setup do
+        #     add_response "kosovo"
+        #   end
+        #   should "take you to other outcome" do
+        #     assert_current_node :esa_going_abroad_eea_outcome
+        #   end
+        # end
```

Comment:
A couple more commented blocks have snuck in. Would be good to remove them and only add them when they're uncommented.

Or in this case, just don't comment them out at all - looks like it wasn't needed, if I've understood 21789ced3f55a85448e23e68fa28e677734e64ee.